apps/mobile/src/components/Feed/EventTokenGrid.tsx

```diff
@@ -115,7 +115,7 @@ export function EventTokenGrid({
 
   return (
     <View
-      className="flex flex-row flex-wrap p-[2]"
```

Comment:
small ask from @kaitoo1 the other day to remove the padding on the outside of the grid.